include/avt_vimba_camera/avt_vimba_camera.h

```diff
@@ -166,6 +167,7 @@ class AvtVimbaCamera
   void updatePixelFormatConfig(Config& config);
```

Comment:
The function is updating the `config` or actually uses info inside `config` to configure the camera?
1. If update the `config`, the argument should be defined as `Config *config`.
2. If the info inside `config` is used to configure the camera, the argument should be `const config&` and the function name is better to be changed into `configPixelFormat(const Config&)`

Reply:
In this situation, config should be passed as const ref. However, should I apply those changes in this PR? Or separate PR should be created?

Reply:
@jilinzhouas , this was the original design, the `config` parameter is both an input and output, since the driver will try to use the value passed in, but also return/update the value that was used by the camera (sometimes they are different).

Reply:
In this case, it should use a pointer instead of a reference even though they both work fine. But we can address it later.